db/access.php

```diff
@@ -35,5 +35,13 @@
                 'archetypes'   => array(
                         'manager' => CAP_ALLOW
                 )
+            ),
```

Comment:
overly pedantic: bracket alignment

Reply:
I have fixed this.

Reply:
I just meant the bracket on the highlighted line was indented one level too many.

I'm happy to go with the four spaces version if everyone else is
@Patches- @lzabo